modules/auxiliary/admin/mssql/mssql_idf.rb

```diff
@@ -104,11 +104,16 @@ def run
     widths = [0, 0, 0, 0, 0, 9]
     total_width = 0
 
-    (column_data|headings).each { |row|
-      0.upto(4) { |col|
-        widths[col] = row[col].length if row[col].length > widths[col]
+    begin
+      (column_data|headings).each { |row|
+        0.upto(4) { |col|
+          widths[col] = row[col].length if row[col].length > widths[col]
+        }
       }
-    }
+    rescue NoMethodError
+      print_error("No columns matched the pattern #{datastore['NAMES'].inspect}. Update the NAMES option to change this search pattern.")
```

Comment:
Would it be cleaner to have an explicit nil check for `column_data` earlier in the process, rather than catching an error after the fact? 👀

Reply:
Yea, I can get that changed to be checked before it reaches that block that throws the error 👍